Tests/RedisKitTests/RedisKitTests.swift

```diff
@@ -14,6 +14,128 @@ final class RedisKitTests: XCTestCase {
         }.wait()
     }
 
+    func testCRUD() throws {
+        try self.client.set("hello", to: "world").wait()
+        let get = try self.client.get("hello", as: String.self).wait()
+        XCTAssertEqual(get, "world")
+        let _ = try self.client.delete(["hello"]).wait()
+        XCTAssertNil(try self.client.get("hello", as: String.self).wait())
+    }
+
+    func testSelect() throws {
+        let _ = try self.client.select(database: 2).wait()
+        try self.client.set("hello", to: "world").wait()
+        let get = try self.client.get("hello", as: String.self).wait()
+        XCTAssertEqual(get, "world")
+
+        let _ = try self.client.select(database: 0).wait()
+        XCTAssertNil(try self.client.get("hello", as: String.self).wait())
+
+        let _ = try self.client.select(database: 2).wait()
+        let reget = try self.client.get("hello", as: String.self).wait()
+        XCTAssertEqual(reget, "world")
+
+        let _ = try self.client.delete(["hello"]).wait()
+        XCTAssertNil(try self.client.get("hello", as: String.self).wait())
+    }
+
+    func testSelectViaConfig() throws {
```

Comment:
This is probably the only unit test we need out of all of these, as the rest should be covered by RediStack